src/routes/v1/authenticatedSites/collectionPages.js

```diff
@@ -4,6 +4,7 @@ import { statsMiddleware } from "@root/middleware/stats"
 
 const Bluebird = require("bluebird")
 const express = require("express")
+const { Base64 } = require("js-base64")
```

Comment:
you might want to rebase interactively to drop @alexanderleegs 's commits

Reply:
I think my rebase went wrong somewhere, fixed!